psiturk/psiturk_shell.py

```diff
@@ -941,60 +944,89 @@ def hit_create(self, numWorkers, reward, duration):
             print '*** duration must be greater than 0'
             return
 
-        # register with the ad server (psiturk.org/ad/register) using POST
-        if os.path.exists('templates/ad.html'):
-            ad_html = open('templates/ad.html').read()
-        else:
-            print '\n'.join(['*****************************',
-                             '  Sorry, there was an error registering ad.',
-                             '  Both ad.html is required to be in the templates folder',
-                             '  of your project so that these Ad can be served!'])
-            return
+        if use_psiturk_ad_server:
```

Comment:
Just a suggestion, but maybe create a separate function like `create_psiturk_ad` that takes takes care of creating the ad, and then call out to the function -- that will make the `hit_create` function a bit less bulky.